src/main/java/de/bonndan/nivio/input/ProcessingChangelog.java

```diff
@@ -0,0 +1,146 @@
+package de.bonndan.nivio.input;
+
+import de.bonndan.nivio.model.Component;
+import de.bonndan.nivio.model.Relation;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
+import org.springframework.util.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+@Schema(description = "Changelog for a single landscape processing run (update of landscape from different sources).")
+public class ProcessingChangelog {
+
+    @Schema(description = "The key FullyQualifiedIdentifier of a component")
+    final Map<String, Entry> changes = new HashMap<>();
```

Comment:
I was wondering, who is reading from the ProcessingChangelog? What is the purpose of the object?

Reply:
It was the first attempt to provide a log to users so that they can track changes.